CHANGELOG.md

```diff
@@ -572,26 +618,33 @@ The Monitoring [CRDs] feature is graduated from Alpha to Beta.
 [Admission webhooks]: https://kubernetes.io/docs/reference/access-authn-authz/extensible-admission-controllers/
 [Traffic Shaping]: https://kubernetes.io/docs/concepts/extend-kubernetes/compute-storage-net/network-plugins/#support-traffic-shaping
 [bandwidth plugin]: https://github.com/containernetworking/plugins/tree/master/plugins/meta/bandwidth
+[IPFIX mediator]: https://tools.ietf.org/html/rfc6183
+[go-ipfix]: https://github.com/vmware/go-ipfix
 
 [0.9.1]: #091---2020-08-21
 [0.9.2]: #092---2020-08-27
 [0.9.3]: #093---2020-09-03
 [0.10.1]: #0101---2020-09-30
 [0.10.2]: #0102---2020-11-11
+[0.11.1]: #0111---2020-11-20
 
 [@AbdYsn]: https://github.com/AbdYsn
 [@abhiraut]: https://github.com/abhiraut
 [@alex-vmw]: https://github.com/alex-vmw
 [@andrewsykim]: https://github.com/andrewsykim
 [@antoninbas]: https://github.com/antoninbas
+[@ceclinux]: https://github.com/ceclinux
+[@dumlutimuralp]: https://github.com/dumlutimuralp
 [@dreamtalen]: https://github.com/dreamtalen
 [@Dyanngg]: https://github.com/Dyanngg
+[@guesslin]: https://github.com/guesslin
```

Comment:
ditto: should be after GraysonWu

Reply:
lol thanks, will fix both - I think I had a brain dead moment there :)